config/default.json

```diff
@@ -1,21 +1,10 @@
 {
   "PORT": 3000,
   "HOSTNAME": "http://ml-api-adapter",
-  "DFSP_URLS": {
-    "dfsp1": {
-      "transfers": {
-        "post": "http://localhost:1080/dfsp1/transfers",
-        "put": "http://localhost:1080/dfsp1/transfers/{{transferId}}",
-        "error": "http://localhost:1080/dfsp1/transfers/{{transferId}}/error"
-      }
-    },
-    "dfsp2": {
-      "transfers": {
-        "post": "http://localhost:1080/dfsp2/transfers",
-        "put": "http://localhost:1080/dfsp2/transfers/{{transferId}}",
-        "error": "http://localhost:1080/dfsp2/transfers/{{transferId}}/error"
-      }
-    }
+  "ENDPOINT_CACHE_URI": "http://localhost:3001/participants/{{fsp}}/endpoints",
```

Comment:
I suggest that this is changed to something more meaningful. What would you suggest naming this if we didn't use a cache?